src/main/proto/th2_grpc_common/common.proto

```diff
@@ -23,7 +23,7 @@ import "google/protobuf/wrappers.proto";
 option java_multiple_files = true;
 option java_package = "com.exactpro.th2.common.grpc";
 
-option go_package = "th2-grpc/th2_grpc_common";
+option go_package = "github.com/th2-net/th2-grpc-common-go/pkg/grpc/th2_grpc_common";
```

Comment:
suggestion: we already have grpc-common in the package (repo name). I don't think we need `grpc/th2_grpc_common` sub-package. Also, since we only have a single file, maybe we don't need `pkg` either. So, the package would be `github.com/th2-net/th2-grpc-common-go`